.github/workflows/rust.yml

```diff
@@ -50,13 +84,17 @@ jobs:
           /tmp/librejson.so
         key: ${{ runner.os }}-redisjson
 
-    - name: Install redis
+    - name: Install DB
       if: steps.cache-redis.outputs.cache-hit != 'true'
       run: |
         sudo apt-get update
-        wget https://github.com/redis/redis/archive/${{ matrix.redis }}.tar.gz;
-        tar -xzvf ${{ matrix.redis }}.tar.gz;
-        pushd redis-${{ matrix.redis }} && BUILD_TLS=yes make && sudo mv src/redis-server src/redis-cli $HOME && popd;
+        wget https://github.com/${{ matrix.config.db-org }}/${{ matrix.config.db-name }}/archive/${{ matrix.config.db-version }}.tar.gz;
+        tar -xzvf ${{ matrix.config.db-version }}.tar.gz;
+        pushd ${{ matrix.config.db-name }}-${{ matrix.config.db-version }} && 
```

Comment:
Hmm. I've apparently been needlessly adding backslashes after `&&` at the end of lines my entire career. :thinking: